lib/screens/home/components/wod_calendar.dart

```diff
@@ -36,9 +36,9 @@ class _WodCalendarState extends State<WodCalendar> {
   Widget build(BuildContext context) {
     mainScreenModel = Provider.of<MainScreenModel>(context);
     return TableCalendar(
-      events: mainScreenModel.wods,
-      onDaySelected: (day, wods, _) {
-        mainScreenModel.whenSelectedDay(day, wods);
+      //events: mainScreenModel.wods,
```

Comment:
Remove this comment please. Let's try to make the code as clean as possible, without unnecessary comments :D